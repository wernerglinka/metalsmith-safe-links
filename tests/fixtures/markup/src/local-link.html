<a href="https://www.potatohead.com/test/testing/">Test link</a>