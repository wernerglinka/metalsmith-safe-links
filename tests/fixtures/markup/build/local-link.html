<html><head></head><body><a href="/test/testing/">Test link</a></body></html>