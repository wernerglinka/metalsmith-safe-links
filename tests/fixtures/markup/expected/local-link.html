<a href="/test/testing/">Test link</a>